app-shell/gulpfile.js

```diff
@@ -0,0 +1,7 @@
+  // Set up ts-node to enable loading of TypeScript files.
+require('ts-node').register({
+  noProject: true,
+});
+
+// Trampoline into gulpfile.ts.
+require('./gulpfile.ts');
```

Comment:
I recently learned that with `ts-node`, you don't need this file anymore. You'll have to delete `declare var require` from gulpfile.ts, but then you can just run `gulp build`.